src/phoenix/experimental/evals/evaluators.py

```diff
@@ -18,19 +18,19 @@ class MapReducer:
     def __init__(
         self,
         model: BaseEvalModel,
-        map_prompt_template: PromptTemplate,
-        reduce_prompt_template: PromptTemplate,
+        map_prompt_template: ClassificationTemplate,
```

Comment:
question: I think the intention here was to leave this super flexible - I think you could in theory map reduce and refine over non classification tasks. I think I get your desire here to make the output deterministic via classification but we might need to relax this. Shouldn't we not touch this file for now?

Reply:
Bump: needs to be mapped back to PromptTemplate